### PR TITLE
Default outbound to true if href is external

### LINKED ISSUE #2: [feature] Default outbound to true if `href` points to an external url
Hey just a small feature request — it would be great if the default for `outbound` was a little smarter, and defaulted to `true` if `href` is an external URL, which the user can easily override with `outbound={false}` to force all links to be same-target.

This would be especially useful if you don't have control over the href, for example in content from a CMS, where you want external links to automatically be outbound.

Happy to submit a PR if you're open to it

### PR DESCRIPTION
**Pull request by madeleineostoja**
Closes #2

### CODE REVIEW

File: src/Link.svelte [hunk 1]
```diff
@@ -21,9 +23,19 @@
...
+
+    if (isExternal && typeof outbound === 'undefined') {
+      outbound = true;
+    }
+  }
+
   $: if (outbound) {
-    target = "_blank";
-    if (rel === undefined) rel = "noopener noreferrer";
+    target = '_blank';
```
Comment (madeleineostoja) [inline src/Link.svelte:37]: Sorry seems my prettier config overrode these double quotes, happy to revert

File: src/Link.svelte [hunk 2]
```diff
@@ -21,9 +23,19 @@
     if (response.ok) fetched.set(href, true);
   }
 
+  $: if (process.browser) {
```
Comment (metonym) [inline src/Link.svelte]: What's the reason for this? Can it be taken out?

EDIT: I'm guessing it's a guard for Sapper?

In my case, I'm unable to run it with a non-SSR Rollup set-up. I wonder if `window.URL` is a better substitute

**Review by metonym — CHANGES_REQUESTED** [review]

File: src/Link.svelte (hunk 2 above)
Comment (madeleineostoja) [reply src/Link.svelte]: Whoops, you're totally correct it's a Sapper thing. Sorry I got muddied up between Svelte and Sapper APIs. `window && window.URL` would be a much better substitute, I'll update it

**Comment by madeleineostoja** [comment]
Updated window check and added a note about new `outbound` behaviour to README

**Review by metonym — APPROVED** [review]
Nice!

**Pull request merged**